common/ferrostar/src/routing_adapters/osrm/snapshots/ferrostar__routing_adapters__osrm__tests__parse_osrm_with_exits.snap

```diff
@@ -129,6 +128,8 @@ snapshot_kind: text
             maneuver_modifier: slight right
             roundabout_exit_degrees: ~
             lane_info: ~
+            exit_numbers:
+              - "10"
```

Comment:
this is the result of the fix. given the input in the sample Valhalla response and others:

```json
"primary": {
                    "type": "off ramp",
                    "modifier": "slight right",
                    "text": "OR 213 South: Oregon City, Molalla",
                    "components": [
                      {
                        "text": "Exit",
                        "type": "exit"
                      },
                      {
                        "text": "10",
                        "type": "exit-number"
                      },
                      {
                        "text": "OR 213 South: Oregon City, Molalla",
                        "type": "text"
                      },
                      {
                        "text": "/",
                        "type": "delimiter"
                      },
                      {
                        "text": "OR 213 South",
                        "type": "text"
                      }
                    ]
                  },
```

this only parses out `exit-number` fields. note that, while I have only seen this in `primary` components in both the Valhalla api responses and Mapbox's API responses, it's theoretically possible in all, so it's returned from all of them.

@ianthetechie do you think this is ok, or should we keep it primary only? if this is okay, should we update `lanes` to not be `sub` only?

Reply:
I've also only seen this in primary, so I guess it's fine as-is for now. We can always revisit later if necessary.